refresh/index.js

```diff
@@ -666,10 +666,17 @@ module.exports = Generator.extend({
   parseFromSwagger: function () {
     if (!this.fromSwagger) return
 
-    return swaggerize.parse(this.fs, this.fromSwagger)
-      .then(response => {
-        this.loadedApi = response.loaded
-        this.parsedSwagger = response.parsed
+    return helpers.loadAsync(this.fromSwagger, this.fs)
+      .then(loaded => {
+        return swaggerize.parse(loaded)
+          .then(response => {
+            this.loadedApi = response.loaded
+            this.parsedSwagger = response.parsed
+          })
+          .catch(err => {
```

Comment:
In reference to earlier comment, here would be a better place to have a swagger specific error message.